docs/tutorial/part-zero/index.md

```diff
@@ -1,194 +1,194 @@
+4. Setelah Anda menginstal Homebrew, ulangi langkah ke-2 untuk memverifikasi.
 
-### Mac Users: install Xcode Command Line Tools
+### Pengguna Mac: instal Baris Perintah Xcode
 
-1. Open your Terminal.
-1. On a Mac, install Xcode Command line tools by running `xcode-select --install`.
-   1. If that fails, download it [directly from the Apple site](https://developer.apple.com/download/more/), after signing-in with Apple developer account
-1. After being prompted to start the installation, you'll be prompted again to accept a software license for the tools to download.
+1. Buka Terminal Anda.
+2. Pada Mac, instal baris perintah Xcode dengan menjalankan `xcode-select --install`.
+   1. Jika gagal, unduh [langsung dari situs Apple](https://developer.apple.com/download/more/), setelah masuk dengan akun pengembang Apple
+3. Setelah diminta untuk memulai proses instalasi, Anda akan diminta untuk menerima lisensi perangkat lunak untuk dapat diunduh.
 
-## ⌚ Install Node.js and npm
+## ⌚ Instal Node.js dan npm
 
-Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer.
+Node.js adalah sebuah lingkungan yang dapat menjalankan kode JavaScript di luar browser web. Gatsby dibangun dengan Node.js. Untuk memulai dan menjalankan Gatsby, Anda harus memiliki Node.js versi terbaru yang diinstal di komputer Anda.
 
-_Note: Gatsby's minimum supported Node.js version is Node 8, but feel free to use a more recent version._
+_Catatan: Versi minimum Node.js yang didukung oleh Gatsby adalah Node 8, akan tetapi jangan ragu untuk menggunakan versi yang lebih baru._
 
-1. Open your Terminal.
-1. Run `brew update` to make sure you have the latest version of Homebrew.
-1. Run this command to install Node and npm in one go: `brew install node`
+1. Buka Terminal Anda.
+2. Jalankan `brew update` untuk memastikan Anda memiliki Homebrew versi terbaru.
+3. Jalankan perintah ini untuk menginstal Node dan npm sekaligus: `brew install node`
 
-Once you have followed the installation steps, make sure everything was installed properly:
+Setelah Anda mengikuti langkah-langkah instalasi, pastikan semuanya telah diinstal dengan benar:
 
-### Check your Node.js installation
+### Periksa instalasi Node.js Anda
 
-1.  Open up your terminal.
-2.  Run `node --version`. (If you’re new to the command line, “run `command`” means “type `node --version` in the command prompt, and hit the Enter key”. From here on, this is what we mean by “run `command`”).
-3.  Run `npm --version`.
+1. Buka Terminal Anda.
+2. Jalankan `node --version`. (Jika Anda baru dalam menggunakan baris perintah, “jalankan `command`” berarti “ketik `node --version` di *command prompt*, dan tekan tombol *Enter*”. Dari sini, inilah yang kami maksud dengan “jalankan `command`”).
```

Comment:
please keep "run" , and dont change it with "jalankan".

Reply:
Run with italic text, right?

Reply:
Since `run` is a verb and not part of command "name" in this context, i just think `jalankan` is easier to understand. But if it's not the case, i'll change all `jalankan` words to `run` since the word is widely used in this file.

Reply:
Hi bangun, sorry my mistake, please ignore this one